### PR TITLE
Cmj.electron builder

### PR DESCRIPTION
**Pull request by ChiaMineJP**
## Caution
This PR must be merged along with this PR (https://github.com/Chia-Network/chia-blockchain/pull/12953).  
See detail on the PR.

### CODE REVIEW

File: packages/gui/package.json [hunk 1]
```diff
@@ -2,13 +2,95 @@
...
+  "license": "Apache-2.0",
   "productName": "Chia Blockchain",
   "private": true,
   "homepage": "./",
   "main": "build/electron/main.js",
   "engines": {
     "node": ">=12.20.0"
   },
+  "build": {
+    "productName": "chia",
```
Comment (paninaro) [inline packages/gui/package.json:14]: Is there a reason we shouldn't capitalize it as "Chia"? The macOS PoC build has the app named "chia.app" when it should be "Chia.app"

File: packages/gui/package.json [hunk 2]
```diff
@@ -2,13 +2,95 @@
...
+      "!public"
+    ],
+    "fileAssociations": [
+      {
+        "ext": "offer",
+        "name": "Chia Offer",
+        "description": "Chia Offer",
+        "role": "Editor"
+      }
+    ],
```
Comment (paninaro) [inline packages/gui/package.json:33]: Can we remove this for now? I know it was in my earlier PR, but we're not quite ready to expose this functionality.

Comment (ChiaMineJP) [reply packages/gui/package.json]: I removed the lines from `packages/gui/package.json`

File: packages/gui/package.json (hunk 1 above)
Comment (ChiaMineJP) [reply packages/gui/package.json:14]: @wallentx 
Any ideas? I cherry-picked the change from this commit (https://github.com/Chia-Network/chia-blockchain-gui/pull/903/commits/7077750fe6fc532c7a17e08dd47bfcf99b15be73)

Comment (paninaro) [reply packages/gui/package.json:14]: Ah ok, maybe this was done for Linux, but on macOS/Windows we should capitalize it. We can address that in the build script I think.

Comment (ChiaMineJP) [reply packages/gui/package.json:14]: Updated mac executable file name to `Chia.app` from `chia.app`.

**Review by paninaro — APPROVED** [review]
lgtm

**Pull request merged**